mPower2/mPower2/View Controllers/TodayViewController.swift

```diff
@@ -523,14 +502,64 @@ class TodayTableViewCell: UITableViewCell {
 }
 
 extension RSDTaskInfoObject {
```

Comment:
Why is this  extension applied to RSDTaskInfoObject whereas the other is applied to RSDTaskInfo?

Reply:
@syoung-smallwisdom not sure, but I removed the one for 'pluralTerm' as it wasn't being used.